app/utils/getNews.ts

```diff
@@ -0,0 +1,16 @@
+import { schemaNewsResponse } from '@/models/schemes';
+import { NewsResponse } from '@/models/types';
+import axios from 'axios';
+
+const getNews = async (keyword: string): Promise<NewsResponse> => {
+  try {
+    const response = await axios.get(`${process.env.URL}/everything?q=${keyword}&apiKey=${process.env.API_KEY}`);
+    schemaNewsResponse.parse(response.data);
+
+    return response.data;
+  } catch (err) {
+    throw err;
```

Comment:
What is going to handle this error? For now you can just remove `throw err;` We’ll handle this error in the future tasks